Library/Formula/libgxps.rb

```diff
@@ -1,8 +1,9 @@
 class Libgxps < Formula
   desc "GObject based library for handling and rendering XPS documents"
   homepage "https://live.gnome.org/libgxps"
-  url "http://ftp.gnome.org/pub/gnome/sources/libgxps/0.2/libgxps-0.2.2.tar.xz"
-  sha256 "39d104739bf0db43905c315de1d8002460f1a098576f4418f69294013a5820be"
+  url "https://download.gnome.org/sources/libgxps/0.2/libgxps-0.2.3.tar.xz"
+  mirror "https://github.com/GNOME/libgxps/archive/0.2.3.tar.gz"
```

Comment:
This mirror has a different `sha256`: `869c8b7ca524b1357c97a7b5b8d9827e04f77a084e6fbb0e6509d8e39f6e4148`.